ffi/blitbuffer.lua

```diff
@@ -2122,6 +2135,21 @@ function BB_mt.__index:paintRoundedRect(x, y, w, h, c, r)
     end
 end
 
+--[[
+paintRoundedRect variant that uses ColorRGB32 instead of a luminance value
```

Comment:
```suggestion
paintRoundedRect variant that uses ColorRGB32 instead of a luminance value
NOTE: Be aware that `paintBorderRGB32` currently does *NOT* support rounded corners, so this implementation is *also* incomplete!
```